cluster/restore-from-backup.sh

```diff
@@ -205,6 +212,13 @@ mv /var/etcd/data "${MNT_DISK}/var/etcd-corrupted"
 # Replace the corrupted data dir with the resotred data.
 mv "${BACKUP_DIR}" /var/etcd/data
 
+if [ "${RESTORE_EVENT_ETCD:-}" == "true" ]; then
```

Comment:
since this doesn't restore, it just resets, maybe we could name this `RESET_EVENT_ETCD` or something that is more closely aligned with the effect.